<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShortcutListDialog</class>
 <widget class="QDialog" name="ShortcutListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="shortcutListLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="margin">
    <number>8</number>
   </property>
   <item>
    <widget class="QTableWidget" name="shortcutListTableWidget">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Action</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Shortcut</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="shortcutLayout">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="defaultRadioButton">
       <property name="text">
        <string>Default: None</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="customShortcutLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <widget class="QRadioButton" name="customRadioButton">
         <property name="text">
          <string>Custom:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="shortcutPushButton">
         <property name="minimumSize">
          <size>
           <width>84</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>None</string>
         </property>
         <property name="icon">
          <iconset resource="../images/openlp-2.qrc">
           <normaloff>:/system/system_settings.png</normaloff>:/system/system_settings.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="clearShortcutToolButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../images/openlp-2.qrc">
           <normaloff>:/system/clear_shortcut.png</normaloff>:/system/clear_shortcut.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="customShortcutSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="shortcutListButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images/openlp-2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>shortcutListButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>ShortcutListDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shortcutListButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>ShortcutListDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
